huntsman_dust/power_spectrum.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import util_rad_profile
+from scipy import fftpack
+from scipy import optimize
+import matplotlib.pyplot as plt
+import huntsman_dust.util_plot as util_plot
+from huntsman_dust.detect_mask_sources import detect_mask_sources
+
+
+def power_spectrum(image):
+    image = image.astype(float)
+    psd2D = np.abs(fftpack.fftshift(fftpack.fft2(image)))**2
+    psd1D = util_rad_profile.azimuthalAverage(psd2D)
+    return psd1D
+
+
+if __name__ == '__main__':
```

Comment:
To test the code using this `__main__` function, I have to edit the image_path directly in the file. Better to call a function named `main` and accept command-line arguments. See for example how this works:
https://github.com/AstroHuntsman/POCS/blob/develop/scripts/collect-grid-stats.py
That way I can specify the local file I want to test by just running:
`python huntsman_dust/power_spectrum.py ngc6822_r.fits`
on the command-line.

This can be done for your `detect_mask_source.py` file too.

Reply:
Done!